### PR TITLE
Ciecie grafik

### PR DESCRIPTION
**Pull request by przemekjuzwiszin**

### CODE REVIEW

**Review by karbowskam — CHANGES_REQUESTED** [review]
I think that's ok! 👍 
But you have typo in footer. In the last sentence, you wrote "tydodni", but I think that you meant "tygodni" :)

File: wizytowka.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+            </p>
+            <footer class="article__footer">
+                <img class="article__picture" src="wizytowka.jpg" alt="Grafika pocięta przez nożyczki">
+                Autorem jest Przemek Juzwiszin.
+            </footer>
+        </article>
+    </main>
+    <footer class="footer">
+        Strona domowa Przemka Juzwiszin, która powstała dzięki udziale w szkoleniu: Co ten frontend? Szkolenie trwa
+        12 tydodni.
```
Comment (karbowskam) [inline wizytowka.html:39]: Here's a typo! :)

**Review by karbowskam — CHANGES_REQUESTED** [review]

**Review by robson60000 — APPROVED** [review]
Looks good to me

File: wizytowka.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+                zadania istotne dla projektu i wysyłam na GitHuba.
+            </p>
+            <footer class="article__footer">
+                <img class="article__picture" src="wizytowka.jpg" alt="Grafika pocięta przez nożyczki">
+                Autorem jest Przemek Juzwiszin.
+            </footer>
+        </article>
+    </main>
+    <footer class="footer">
+        Strona domowa Przemka Juzwiszin, która powstała dzięki udziale w szkoleniu: Co ten frontend? Szkolenie trwa
```
Comment (karbowskam) [inline wizytowka.html:38]: Here's another typo! :)

**Review by karbowskam — CHANGES_REQUESTED** [review]

**Review by robson60000 — APPROVED** [review]

**Pull request merged**